### PR TITLE
Fixes #33123 - Add GraphQL type for permissions

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
[test upgrade]

File: app/graphql/types/user.rb [hunk 1]
```diff
@@ -20,5 +20,6 @@ class User < BaseObject
     has_many :personal_access_tokens, Types::PersonalAccessToken
     has_many :ssh_keys, Types::SshKey
     has_many :usergroups, Types::Usergroup
+    has_many :permissions, Types::Permission
```
Comment (ezr-ondrej) [inline app/graphql/types/user.rb:23]: Would it be possible to add this to the user query test?
Not a blocker, tho :)

**Review by ezr-ondrej — DISMISSED** [review]

File: app/graphql/types/user.rb (hunk 1 above)
Comment (xprazak2) [reply app/graphql/types/user.rb:23]: Added to user query test.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @xprazak2 ! :+1:

**Comment by ezr-ondrej** [comment]
Should be safe, will merge when Jenkins approves

**Pull request merged**